src/components/header/ACLK/use-cloud-connection-status.tsx

```diff
@@ -0,0 +1,69 @@
+import React, { useMemo } from "react"
+import { CloudConnectionProps, ConnectionModalStatusContent } from "./types"
+import Anchor from "@/src/components/anchor"
+
+import { Text } from "@netdata/netdata-ui"
+
+export const makeCloudConnectionStatusInfo = ({
+  nodeStatus,
+  userStatus,
+  date,
+}: CloudConnectionProps): ConnectionModalStatusContent => ({
+  title: "Netdata Cloud connection status",
+  text: {
+    header: () => {
+      return (
+        <Text>
+          This node is currently{" "}
+          <Text strong>{nodeStatus === "LIVE" ? "Connected" : "Not Connected"}</Text> to Netdata
+          Cloud
+        </Text>
+      )
+    },
+    bullets:
+      nodeStatus === "NOT_LIVE"
+        ? [
+            // `The node lost its Netdata Cloud connection at ${date}`,
+            () => (
+              <Text>
+                To troubleshoot Netdata Cloud connection issues, please follow{" "}
+                <Anchor
+                  target="_blank"
+                  rel="noopener noreferrer"
+                  href="https://learn.netdata.cloud/docs/agent/claim#troubleshooting"
+                >
+                  this guide
+                </Anchor>
+                .
+              </Text>
+            ),
+          ]
+        : [],
+    footer: () => (
+      <Text>
+        You are{" "}
+        <Text strong>
+          {userStatus === "LOGGED_IN"
+            ? "Logged In"
+            : userStatus === "EXPIRED_LOGIN"
+            ? "Logged out"
+            : "Not signed-up"}
```

Comment:
Maybe it would be better to have the statuses in a mapping object
```
const userStatusTexts = {
  LOGGED_IN: "Logged In",
  EXPIRED_LOGIN: "Logged out"
  DEFAULT: "Not signed-up"
}
```
and use them as:

```
userStatusTexts[userStatus] || userStatusTexts.DEFAULT
```

I think it would be more readable this way! 😄

Reply:
I will do this way! Thank you